### DISCUSSION TITLE
Clarify timezone of pushes/releases and make them consistent

### DISCUSSION THREAD

**Discussion by Philipp91** [npm/feedback#143] [Website] [answered]
The typescript package, for example, had its [latest nightly release](https://www.npmjs.com/package/typescript/v/4.2.0-dev.20210108) (as of right now) about "a day ago". This information is present both at the top of the page ("Published a day ago") and in the right column (under the "Last publish" header). Both "ago" fields show a timestamp on hover, but without telling the user what the timestamp's timezone is. It would be great to add the timezone, because it helps figuring out which code (from the Git repo) is inside that release.

So formulated as a feature request: Please print the timezone in these hover titles. Maybe make them all UTC and append "UTC".

----------

The right timezone can be inferred manually because they're actually inconsistent between what looks like server-side rendering and client-side rendering. The timestamp in the right column is never updated (because no matter which version I view, the "last publish" info for the package isn't version-dependent). It's always this:

```
<time datetime="2021-01-08T07:35:13.880Z" title="2021-1-8 07:35:13">a day ago</time>
```

This is in UTC and with an English/international date format, as one would expect from server-side rendering.

The top "Published" information tells when the version was published that is currently being viewed. Upon page load, it's the same. But when I navigate away from that version and back (e.g. click on the "Versions" tab and then select the same version again), then I get this:

```
<time datetime="2021-01-08T07:35:13.880Z" title="8.1.2021, 08:35:13">a day ago</time>`
```

This is the formatting for my local timezone (UTC+1) and locale (de-DE).

So formulated as a feature request: Either display all timestamps with UTC (formatted according to ISO 8601) or display them all in the local timezone with the user's locale. Personally, I prefer the former, and it also seems easier to implement.

**Comment by MylesBorins** [comment, marked as answer]
Hey @Philipp91,

I've opened a ticket with our engineering team and I'll keep you up to date. Thanks for reporting

**Reply by MylesBorins** [reply]
@Philipp91 thanks for submitting. At this time we are not going to be able to prioritize fixing this. We are tracking the bug in our backlog and may revisit in the future.
